python/pyiceberg/cli/console.py

```diff
@@ -17,24 +17,18 @@
 # pylint: disable=broad-except,redefined-builtin,redefined-outer-name
 from functools import wraps
 from typing import (
-    Dict,
     Literal,
     Optional,
     Tuple,
-    Type,
 )
 
 import click
 from click import Context
 
 from pyiceberg.catalog import Catalog, load_catalog
-from pyiceberg.catalog.hive import HiveCatalog
-from pyiceberg.catalog.rest import RestCatalog
 from pyiceberg.cli.output import ConsoleOutput, JsonOutput, Output
 from pyiceberg.exceptions import NoSuchNamespaceError, NoSuchPropertyException, NoSuchTableError
 
-SUPPORTED_CATALOGS: Dict[str, Type[Catalog]] = {"thrift": HiveCatalog, "http": RestCatalog}
```

Comment:
This was still lingering here, but isn't needed and causes issues when thrift isn't installed.